update.php

```diff
@@ -408,3 +408,25 @@ function update_1327()
 	return Update::SUCCESS;
 }
 
+function update_1329()
+{
+	$currStorage = Config::get('storage', 'class', '');
+
+	if (!empty($currStorage)) {
+		$storageName = array_key_first(\Friendica\Core\StorageManager::DEFAULT_BACKENDS, $currStorage);
+		Config::set('storage', 'name', $storageName);
+		Config::delete('storage', 'class');
+	}
+
+	$photos = DBA::select('photos', ['backend-class', 'id'], ['backend-class IS NOT NULL']);
+	foreach ($photos as $photo) {
+		DBA::update('photos', ['backend-class' => $photo['backend-class']::NAME], ['id' => $photo['id']]);
```

Comment:
Like above.

Could you explain, what this line is supposed to do?

I also see this functionality very critically. For example the "photo" table on my systems range from 1 million to 2 million entries. This update routine would result in a downtime of several hours, possibly even days. Please evaluate how this could be done in a single query or some "post update" functionality.